packages/api-reference/src/features/Operation/components/RequestBody.vue

```diff
@@ -4,54 +4,54 @@ import { ScalarMarkdown } from '@scalar/components'
 import type { ContentType, RequestBody } from '@scalar/types/legacy'
 import { computed, ref } from 'vue'
 
-const prop = defineProps<{ requestBody?: RequestBody }>()
+const props = defineProps<{ requestBody?: RequestBody }>()
```

Comment:
```suggestion
const { requestBody } = defineProps<{ requestBody?: RequestBody }>()
```
remember we can destructure props now! if you like